godot-macros/src/class/data_models/interface_trait_impl.rs

```diff
@@ -204,6 +210,76 @@ pub fn transform_trait_impl(original_impl: venial::Impl) -> ParseResult<TokenStr
                 });
             }
 
+            #[cfg(before_api = "4.3")]
+            "get_property_list" => {
+                get_property_list_impl = quote! {
+                    #(#cfg_attrs)*
+                    compile_error!("`get_property_list` is only supported for Godot versions of at least 4.3");
+                };
+
+                // Set these variables otherwise rust complains that these variables arent changed in Godot < 4.3.
+                get_property_list_fn = None;
+                free_property_list_fn = None;
+            }
+
+            #[cfg(since_api = "4.3")]
+            "get_property_list" => {
+                get_property_list_impl = quote! {
+                    #(#cfg_attrs)*
+                    impl ::godot::obj::cap::GodotGetPropertyList for #class_name {
+                        fn __godot_get_property_list(&mut self) -> Vec<::godot::builtin::meta::PropertyInfo> {
+                            // Only supported in godot api > 4.3. If support is added for earlier versions this is still needed.
```

Comment:
You probably mean `>= 4.3` here and not `>`.

Also not exactly sure what you mean by "If support is added for earlier versions this is still needed.", could you maybe elaborate what you mean by "support" and "this is still needed"?

(also capitalization, "Godot API")